src/main/java/com/ss/ita/rozetka/pageobject/elements/Product.java

```diff
@@ -0,0 +1,116 @@
+package com.ss.ita.rozetka.pageobject.elements;
+
+import io.qameta.allure.Step;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.support.Color;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.codeborne.selenide.Selenide.*;
+
+@RequiredArgsConstructor
+public class Product {
+
+    @NonNull
+    private final String xPath;
+    private String promoLabelTitle;
+    private String productTitle;
+    private BigDecimal productOldPrice;
+    private BigDecimal productPrice;
+    private List<String> availableColors;
+    private String availability;
+    private int amountReviews;
+    private String productDescription;
+
+    @Step("Product: get product title")
+    public String getProductTitle() {
+        productTitle = $x(String.format("%s%s", xPath, "//span[@class='goods-tile__title']")).text();
+        return productTitle;
+    }
+
+    @Step("Product: get promo label title")
+    public String getPromoLabelTitle() {
+        try {
+            promoLabelTitle = $x(String.format("%s%s", xPath, "//span[contains(@class,'goods-tile__label')]")).text();
+        } catch (AssertionError exception) {
+            promoLabelTitle = StringUtils.EMPTY;
+        }
```

Comment:
do not use try..catch to check simple conditions
use if el.exist() then return elemen title else return empty

Reply:
same for all